cip/CIP2016-06-14-Harmonize-comparability-orderability-equality-and-equivalence.adoc

```diff
@@ -0,0 +1,233 @@
+= CIP2016-06-14 - Harmonize Comparability, Orderability, Equality, and Equivalence
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Mats Rydberg <mats@neotechnology.com>, Stefan Plantikow <stefan.plantikow@neotechnology.com>
+
+[abstract]
+.Abstract
+--
+This CIP intends to introduce and formalise two new concepts, *comparability* and *orderability*, and defines their relationship to the already existing concepts of *equality* and *equivalence*.
```

Comment:
"already existing" -> "pre-existing"
I see all 4 concepts are covered in this CIP... so was wondering how `equality` and `equivalence` are pre-existing? Are they in another CIP or defined somewhere else? If yes, this other resource ought to be linked here...